jupyter/Notebook Binding/Numeric Tile.ipynb

```diff
@@ -0,0 +1,66 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<table>\n",
+    "    <tr>\n",
+    "        <td><img src='../SystemLink_icon.png' /></td>\n",
+    "    </tr>\n",
+    "</table>\n",
+    "\n",
+    "## Numeric Tile Binding Example\n",
+    "***\n",
+    "This example illustrates how a notebook execution can be bound to numeric tile in a dashboard builder.\n",
```

Comment:
"a dashboard builder" should be "Dashboard Builder", this is in the other notebooks as well.